src/gpu/nvidia/cudnn_binary.cpp

```diff
@@ -33,19 +36,31 @@ status_t cudnn_binary_t::execute(const exec_ctx_t &ctx) const {
             = utils::downcast<nvidia::sycl_cuda_stream_t *>(ctx.stream());
 
     return cuda_stream->interop_task([&](::sycl::handler &cgh) {
-        auto src_0_acc = CTX_IN_ACCESSOR(DNNL_ARG_SRC_0);
-        auto src_1_acc = CTX_IN_ACCESSOR(DNNL_ARG_SRC_1);
-        auto dst_acc = CTX_OUT_ACCESSOR(DNNL_ARG_DST);
+        auto *mem_src_0 = static_cast<sycl::sycl_memory_storage_base_t *>(
```

Comment:
Could the logic be kept inside a single macro to avoid code bloating for each and every implementation, please? This doesn't look like a scalable solution to me as of now.

Reply:
done